### DISCUSSION TITLE
zigbee2mqtt process memory footprint - increases over time ? (memory leaks?)

### DISCUSSION THREAD

**Discussion by RobLeighton67** [Koenkk/zigbee2mqtt#11010] [General]
I'm using 1.22.2 via dietpi linux on a Pi Zero - this uses node.js 11.15 (the last supported armv6l version)

My installation starts with the node index.js process consuming about 60Mb , but after running for 6 hours the process (according to htop) has a footprint of 75Mb.

I'm looking to have my installation run 24/7 - and am not really wanting to restart periodically to avoid running out of resource.

**Comment by RobLeighton67** [comment]
Just to add, using 5 sensor devices on a (latest firmware) CC2531 - not a heavily loaded system - the increase occurs even with my supporting application (that communicates via MQTT) disabled.

**Comment by bjohas** [comment]
Hi @RobLeighton67 - how did this go? We're thinking of running zigbee2mqtt on Raspberry Pi Zero. Any experiences you can share?

**Reply by RobLeighton67** [reply]
Sorry, just noticed this reply - it works for me, no GUI, everything stripped back and some minor customisation.
The memory footprint thing has not been an issue - it tops out at a certain value (seemingly proportional to the number of sensors attached).
